UI/ModsMenu/ModsMenu.tscn

```diff
@@ -103,13 +90,12 @@ margin_left = -300.0
 margin_top = -200.0
 margin_right = 300.0
 margin_bottom = 200.0
-window_title = "Open a File"
+rect_min_size = Vector2( 150, 52.5 )
+window_title = "Открыть файл"
```

Comment:
Meow?

Reply:
Woops... Looks like in dialog popup has enabled "rename_title" option, what changes title name to my native language.